bridges/otelzap/core_test.go

```diff
@@ -31,6 +33,29 @@ func TestCore(t *testing.T) {
 	assert.Equal(t, log.SeverityInfo, got.Severity())
 }
 
+func TestCoreEnabled(t *testing.T) {
+	enabledFunc := func(c context.Context, r log.Record) bool {
+		return r.Severity() >= log.SeverityInfo
+	}
+
+	r := logtest.NewRecorder(logtest.WithEnabledFunc(enabledFunc))
+	logger := zap.New(NewCore(WithLoggerProvider(r)))
+
+	if ce := logger.Check(zap.DebugLevel, testMessage); ce != nil {
+		ce.Write()
+	}
+
+	assert.Nil(t, r.Result()[1].Records)
+
+	if ce := logger.Check(zap.InfoLevel, testMessage); ce != nil {
+		ce.Write()
+	}
+
+	got := r.Result()[1].Records[0]
```

Comment:
let's make sure that this test will not panic if the implementation does not work properly

```suggestion
	require.Len(t, r.Result()[1].Records, 1)
	got := r.Result()[1].Records[0]
```